doc/admin-tasks.md

```diff
@@ -0,0 +1,11 @@
+## Admin Tasks
```

Comment:
@leenagupte Ah sure - makes sense (especially given the acceptance criteria) - I don't think it's too pressing for this to be available to 2nd line. I've no memory of this ever coming up as an issue since I've been here, so probably more a specialist task. Guess it'd be nicer if we had a way of documenting the rake tasks automatically for documentation, but not sure how to do that.

Seems cool though if you think it'll be useful, can just re-asses later if it becomes a pain. Would you be able to include documentation for represent_downstream too since that's a set of common tasks?